nav2_behavior_tree/include/nav2_behavior_tree/bt_action_server_impl.hpp

```diff
@@ -59,6 +59,15 @@ BtActionServer<ActionT>::BtActionServer(
   if (!node->has_parameter("default_server_timeout")) {
     node->declare_parameter("default_server_timeout", 20);
   }
+
+  std::vector<std::string> error_code_id_names = {
+    "follow_path_error_code_id",
+    "compute_path_error_code_id"
+  };
+
+  if (!node->has_parameter("error_code_id_names")) {
+    node->declare_parameter("error_code_id_names", error_code_id_names);
```

Comment:
I want to do something like this.

`#include<iostream>
#include<vector>
template <typename T>
std::ostream & operator << (std::ostream & os, const std::vector<T> & vec)
{
    for(auto elem : vec)
    {
        os<<elem<< " ";
    }
    return os;
}`

I am not really sure where it would go... Could I add it to the nav2_utils?

Reply:
Sure thing